usaspending_api/broker/management/commands/load_broker_static_data.py

```diff
@@ -30,4 +31,19 @@ def handle(self, *args, **options):
                 new_rows += 1
 
         logger.info("Stored External Data Types, {} records added".format(new_rows))
+
+        # Delta Table Load Version
+        logger.info("Loading Delta Table Load Versions...")
+        new_rows = 0
+
+        for load_version_type in lookups.DELTA_TABLE_LOAD_VERSION_TYPE:
+            _, created = DeltaTableLoadVersion.objects.update_or_create(
+                delta_table_load_version_id=load_version_type.id,
+                name=load_version_type.name,
+                defaults={"description": load_version_type.desc},
+            )
+            if created:
```

Comment:
Is `created` a Boolean? If not I think this if statement could be made more clear.

Reply:
Yes, this is a boolean